src/NewsletterEpic/index.tsx

```diff
@@ -169,21 +174,24 @@ const CTA: React.FC<CTAProps> = (props: CTAProps) => {
 
                             subscribeToNewsletter(newsletterId as string)
                                 .then(() => setSubscribeClickStatus('SUCCESS'))
-                                .catch(() => setSubscribeClickStatus('FAILURE'));
+                                .catch(() => {
+                                    setSubscribeClickStatus('FAILURE');
+                                    setTimeout(() => setSubscribeClickStatus('DEFAULT'), 3500);
+                                });
                         }}
                     >
                         Sign up
                     </Button>
                 </ThemeProvider>
             );
-        case 'IN_PROGRESS':
+        case 'FAILURE':
             return (
-                <ThemeProvider theme={buttonBrandAlt}>
-                    <Button css={ctaStyles.button} disabled={true}>
-                        Loading...
-                    </Button>
-                </ThemeProvider>
+                <div css={ctaStyles.errorText}>
+                    There was an error signing up to the newsletter. Please try agian
```

Comment:
Here we're asking the user to try again, but it doesn't look like we're giving them an easy way to do that, should we re-render the button? Or make `Try again` clickable?

Reply:
I've gone for re-render the sign up button and moved that button into it's own component. Do you reckon we're okay with the amount this changes the height of the epic though?

Reply:
Good question. All else being equal I guess it'd be nice if it didn't increase the height too much but since we expect this error case to be fairly exceptional I don't think we should worry too much.